### ISSUE TITLE
Not displaying correct audio wave on oreo, nougat and marshmallow

### ISSUE THREAD

**Issue by JasonTheDynamite** [alxrm/audiowave-progressbar#13]
Unfortunately this library does not give correct audio wave on marshmallow, nougat and oreo. When I input a file of length 2 seconds and sound starts at 0:00 and ends at 00:01, rest seconds are blank.. then the audio wave shows no blank moments. Instead it shouw as if the there was sound during the whole recording.

Proof:
http://yarmobile.nazwa.pl/PROJEKTY/WeType/20180511_164630.mp4
http://yarmobile.nazwa.pl/PROJEKTY/WeType/20180511_162830.mp4

**Comment by alxrm** [comment]
@reckos yes it doesn't, read these comments [here](https://github.com/alxrm/audiowave-progressbar/issues/10#issuecomment-351470131) and [here](https://github.com/alxrm/audiowave-progressbar/issues/1#issuecomment-265679117)

**Comment by JasonTheDynamite** [comment]
will you fix this ??

**Comment by alxrm** [comment]
@reckos I don't have time to do that at the moment, sorry for inconvenience
